scripts/test-downgrade-protection.py

```diff
@@ -86,6 +86,10 @@ def main():
     else:
         run_only = None
 
+    if srv_max_prot is None:
+        raise ValueError("Highest protocol version supported by server unset,"
+                         " use --server-max-protocol to set it")
+
```

Comment:
why this change ?
The semantic here was that an unspecified max prot means the maximum we support (TLS 1.3)

Reply:
it mostly is a leftover when I was trying to figure out why the gnutls bug wasn't being detected, it indeed is unnecessary now, I'll drop that patch